src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1630,6 +1619,10 @@ void TorrentHandle::handleMetadataReceivedAlert(libt::metadata_received_alert *p
         m_speedMonitor.reset();
         m_session->handleTorrentPaused(this);
     }
+    
+    // If first/last piece priority was specified when adding this torrent, set it up now that we can:
+    if (m_hasFirstLastPiecePriority)
+        setFirstLastPiecePriority(true);
```

Comment:
@glassez I discovered that, similar to first/last piece priority, if a magnet link is added with sequential download but qBittorrent quits before its metadata is retrieved, then `sequential` is not saved. So I added some code to save `sequential` in the resume data when a torrent's metadata isn't retrieved yet to fix the bug.